components/analytics/system_analytics/system_analytics.jsx

```diff
@@ -362,12 +364,10 @@ export default class SystemAnalytics extends React.PureComponent {
 
         return (
             <div className='wrapper--fixed team_statistics'>
-                <h3 className='admin-console-header'>
-                    <FormattedMessage
-                        id='analytics.system.title'
-                        defaultMessage='System Statistics'
-                    />
-                </h3>
+                <FormattedAdminHeader
```

Comment:
This "breaks" the translation extraction mechanism. There are solutions. The simplest one would be add the `t` function call in the id attribute, something like:
```
<FormattedAdminHeader
  id={t('analytics.system.title')}
  defaultMessage='System Statistics'
/>
```

Other solution is to add the new `FormattedAdminHeader` to the extraction tool, but if you add a lot of this, you will need to add each one to the translations extraction tool.

Other option is try to make the tool smarter, something like try to extract from any component with name starting by "Formatted" with attributes `id` and `defaultMessage`, and if is needed, add specific cases for specific components.

Reply:
I think for now I'm going to go for the second option: adding `FormattedAdminHeader` to the i18n extraction script. If more components like this get added, we can consider the third option then. I don't think the first option is acceptable.